static/css/profile.css

```diff
@@ -42,8 +41,17 @@ main > header {
 .profile-intro section {
   flex-basis: 100%;
   order: 2;
+  margin-top: 5rem;
```

Comment:
I suppose this [substitutes the padding-top](https://github.com/neomad-team/neomad.org/pull/52/files#diff-09e67392be1aad6cabe218beb4b5b83eL13)
TODO